training/model_management/components/model_sanity_validation/spec.yaml

```diff
@@ -0,0 +1,43 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+
+name: mlflow_model_local_validator
+version: 0.0.1
+type: command
+
+is_deterministic: True
+
+display_name: MLFlow model local validator (Public Preview)
```

Comment:
can we just use tags? We have "Preview", "Public preview" and "Public Preview"

Reply:
@vizhur can you please share a tags example? 
`preview: True`

Reply:
Like this: https://github.com/Azure/azureml-assets/blob/main/training/general/src/environments/pytorch-1.11-ubuntu20.04-py38-cuda11-gpu/spec.yaml#L20, which translated to a "Preview" box in the UI.

Reply:
Done. Thanks @elesel